src/app/trainee/revalidation-history.interface.ts

```diff
@@ -0,0 +1,29 @@
+export interface IRevalidationHistory {
+  gmcId: string;
+  traineeName: string;
```

Comment:
think i recall @jayanta2018 saying `traineeName` wont be included but not sure if he meant the property wont be there or if the value will be blank?

Reply:
Yeah, Uzair said he already has the First and Last name and will pass and concatenated Trainee Name